collateral-info.yaml

```diff
@@ -285,6 +292,38 @@ collateral-defs:
     collateral-symbol: RWA009
     collateral-info-url: https://manual.makerdao.com/collateral/real-world-assets-in-detail/hvbank
 
+  RWA010: &RWA010
+    <<: *CFG-DROP
+    collateral-desc: Revolving Credit Facility formed of the following collateral types: Senior Secured Facilities, Forward-Flow Arrangements, Whole loans or receivables purchases, Investment Grade Structured Credit Products (ABS bonds).
+    collateral-longname: Centrifuge Blocktower Credit Series 1 DROP Token
+    collateral-shortname: BlockTower Series 1 DROP
+    collateral-symbol: BBT1DRP
```

Comment:
Did we actually use this as a collateral symbol? If yes, should stick with this, if no, we should maintain the existing convention: BBT1-DROP, etc.

Reply:
the collateral is actually RWA010 in the maker protocol, so I keep the BBT1-DROP recommendation.